test/catch2/live_restore/unit/test_live_restore_extent_list.cpp

```diff
@@ -0,0 +1,119 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * Tests of the Live Restore extent lists. Extent lists track "holes" in a file representing ranges
```

Comment:
Try and copy the formatting in the block/ tests:
```
/*
 * [block_addr]: block_addr.c
 * This file unit tests block manager functions relating to packing and unpacking address cookies.
 */
```

Reply:
[3b7771b](https://github.com/wiredtiger/wiredtiger/pull/11332/commits/3b7771b399d3c9323c6d67572df9faf8a7eb1f5e)